Makefile

```diff
@@ -127,7 +130,7 @@ all: framework
 lib : ${OBJS}
 	@$(ECHOE) ${GREEN}Generating lib [${YELLOW}$(LIB_DIR)/$(TARGET_LIB)${GREEN}]${NC}
 	@$(MKDIR_P) $(LIB_DIR)
-	@$(CC) $(CFLAGS) -o $(LIB_DIR)/$(TARGET_LIB) $^ $(XLDFLAGS)
+	@$(CC) $(XCFLAGS) -o $(LIB_DIR)/$(TARGET_LIB) $^ $(XLDFLAGS)
 
 # Make any c source
 $(BUILD_DIR)/%.o: %.c
```

Comment:
Should we also change line: 151, for XCFLAGS?

@$(ECHOE) ${YELLOW}CFLAGS:${NC} ${CFLAGS}

Reply:
we should still be printing CFLAGS so we know what it is.